torch/_utils.py

```diff
@@ -67,6 +67,17 @@ def _to(self, device, non_blocking=False):
     if self.device == device:
         return self
 
+    if device.type == "cpu":
+        pin_memory = non_blocking and self.device.type in (
+            "cuda",
+            torch._C._get_privateuse1_backend_name(),
+        )
```

Comment:
Following convention for tensors here https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/TensorConversions.cpp?fbclid=IwZXh0bgNhZW0CMTEAAR2nGqQndR7ZZk_X1azbVlPJHPXo9WbD1nA6e63VoOC6VGZSlsziYFJPjVk_aem_2ONazxgggrfNnWlfC6JIqA#L341-L342